examples/shell/mbed/main.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <lib/shell/shell.h>
+
+#include <lib/core/CHIPCore.h>
+#include <lib/support/Base64.h>
+#include <lib/support/CHIPArgParser.hpp>
+#include <lib/support/CodeUtils.h>
+#include <lib/support/RandUtils.h>
+#include <support/logging/CHIPLogging.h>
+
+#include <ChipShellCollection.h>
+
+using namespace chip;
+using namespace chip::Shell;
+
+int main()
+{
+    // Initialize the default streamer that was linked.
+    const int rc = streamer_init(streamer_get());
+
+    if (rc != 0)
+    {
+        ChipLogError(Shell, "Streamer initialization failed: %d", rc);
+        return rc;
+    }
+
```

Comment:
Should we enable the RTC in this function with `set_time(0)` ?

Reply:
It looks like Mbed-Os will automatically execute `set_time(0)`  if the RTC is uninitialized and we call `gettimeofday()`function. Here is [source](https://github.com/ARMmbed/mbed-os/blob/4c4295225ec17f9485c61c72588a02f0d35d6b31/platform/source/mbed_rtc_time.cpp#L100).

Reply:
Good point.